cluster/gce/windows/configure.ps1

```diff
@@ -84,6 +84,20 @@ function FetchAndImport-ModuleFromMetadata {
   Import-Module -Force C:\$Filename
 }
 
+# Returns true if the ENABLE_NODE_LOGGING field in kube_env is true.
+# $KubeEnv is a hash table containing the kube-env metadata keys+values.
+function IsLoggingEnabled {
+  param (
+    [parameter(Mandatory=$true)] [hashtable]$KubeEnv
+  )
+
+  if ($KubeEnv.Contains('ENABLE_NODE_LOGGING') -and `
+      ($KubeEnv['ENABLE_NODE_LOGGING'] -eq 'true')) {
+    return $true
```

Comment:
Could this just be?:

```
  return ($KubeEnv.Contains('ENABLE_NODE_LOGGING') -and `
     ($KubeEnv['ENABLE_NODE_LOGGING'] -eq 'true'))
```

I see this is imitating code that we already have; I'm mostly just curious, if you don't want to bother trying this that's fine.

Reply:
Yeah this is imitating the existing code and thanks for the suggestion! I've added a new kube_env `ENABLE_STACKDRIVER_WINDOWS` as the `ENABLE_NODE_LOGGING` will be deprecated soon. And I just kept them in the if-elseif.